e2e/workspaces/demo_app/fail_fast.yaml

```
@@ -4,6 +4,12 @@ tags:
 ---
 - launchApp:
     clearState: true
-- assertTrue:
-    condition: ${ false }
-    label: Fail the flow
+- evalScript: ${output.didFail = 1}
+- runFlow:
+    optional: true
+    commands:
+        - assertTrue:
+            condition: ${ false }
+            label: Fail the flow
+        - evalScript: ${output.didFail = 0} # This should never run
+- evalScript: ${output.didFail = 1}
```

This last line needs to be some kind of assertion
```suggestion
- assertTrue: ${output.didFail == 1}
```

---

Needs to check that the line that should never run didn't run